routes/create-account.js

```diff
@@ -8,39 +8,57 @@ router.get('/', function(req, res, next) {
 });
 
 router.post('/', async function(req, res, next) {
-    // TODO: Validate input.
-    let email = req.body.email;
+    let email = req.body.email;  // Bootstrap form already validates FORM of email.
     let password = req.body.password;
     let firstname = req.body.firstname;
     let lastname = req.body.lastname;
     let instructor = (req.body.userradios === 'instructor');
 
-    db.User.create({
-        email: email,
-        password: password,
-        name: {first: firstname, last: lastname},
-        instructor: instructor
-    }, async (err, user) => {
+    // Create new User.
```

Comment:
It seems like you need to verify a person's email whenever a user is added to the db. Is there a way to put this verification code into User middleware instead?

Reply:
This comment is vague because some changes might need to be made to the design of how you're creating users in this commit.